### PR TITLE
apps/readme.md: update documentation 

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

**Review by rbudai98 — APPROVED** [review]

File: apps/readme.md [hunk 1]
```diff
@@ -8,3 +8,92 @@ This directory contains different apps that are shipped with the SDK.
...
+The server app is enabled by default. The app and the mass storage can be enabled by editing the usb-gadget.sh file located in
+/usr/share/systemd/usb-gadget.sh. 
+In order to enable this functionality you must uncomment the following lines:
+````
+create_rndis configs/c.1 rndis.0
+create_msd configs/c.1 mass_storage.0 /dev/mmcblk1p1
+````
+If you want to disable this service the lines must be commented.
+````
+#create_rndis configs/c.1 rndis.0
```
Comment (dNechita) [inline apps/readme.md:28]: this line will disable the server app from starting after system boots?

File: apps/readme.md [hunk 2]
```diff
@@ -8,3 +8,92 @@ This directory contains different apps that are shipped with the SDK.
...
+sudo systemctl stop network-gadget
+sudo systemctl stop uvc-gadget
+````
+Copy the apps to /usr/share/systemd:
+```` 
+sudo cp /home/analog/Workspace/ToF/build/apps/server/aditof-server /usr/share/systemd/
+sudo cp /home/analog/Workspace/ToF/build/apps/uvc-app/uvc-app /usr/share/systemd/
+````
+
+After this operation a reboot is required.
```
Comment (dNechita) [inline apps/readme.md:99]: you could show how, e.g.:
sudo reboot

File: apps/readme.md (hunk 1 above)
Comment (SeptimiuVana) [reply apps/readme.md:38]: No, it's for network over usb and maass storage. Updated.

File: apps/readme.md (hunk 2 above)
Comment (SeptimiuVana) [reply apps/readme.md]: Fixed.

**Pull request merged**